### ISSUE TITLE
special case rpb1

### ISSUE THREAD

**Issue by ValWood** [pombase/website#2078]
http://dev.pombase.kmr.nz/gene_protein_features/SPBC28F2.12

we have special allele naming syntax for the rpb1  (and another protein?)
@manulera  can poit you to this. These will be 'conjoined' alleles

(not necessary for release)

**Comment by kimrutherford** [comment]
> we have special allele naming syntax for the rpb1 (and another protein?)

The allele parsing for amino_acid_mutation is very simple at the moment.  There are quite a few that are ignored.  I've been meaning to chat about it on a call.

 rpb1 has descriptions like these that are ignored:
   - https://www.pombase.org/genotype/rpb1-CTD-S5.S5A-%28CTD%29S5A%28alternating%29-amino_acid_mutation-expression-not_assayed_or_wild_type
   - https://www.pombase.org/genotype/rpb1-CTD-S2A-%28CTD%29S2A-amino_acid_mutation-expression-not_assayed_or_wild_type
   - https://www.pombase.org/genotype/rpb1-CTD-S2A%285-16%29,CTDdelta%2817-29%29-CTD-S2A%285-16%29,CTDdelta%2817-29%29-amino_acid_deletion_and_mutation-expression-not_assayed_or_wild_type

spt5 also has some that are ignored:
 - https://www.pombase.org/genotype/spt5-%28CTD%29T1A%287%29-%28CTD%29T1A%287%29-amino_acid_mutation-expression-not_assayed_or_wild_type
 - https://www.pombase.org/genotype/spt5-%28T1A%297-CTD-T1A-amino_acid_mutation-expression-not_assayed_or_wild_type

**Comment by manulera** [comment]
Yes, rpb1 and spt5 are the CTD ones, for now those should be ignored.

**Comment by kimrutherford** [comment]
Is this covered by?:
  - pombase/pombase-chado#1141

**Comment by ValWood** [comment]
Similar, this ticket is for the alleles ...

**Comment by ValWood** [comment]
do not do until
https://github.com/pombase/pombase-chado/issues/1192

**Comment by kimrutherford** [comment]
The seems a bit more challenging than the modifications because of alleles like this:
 - [rpb1-CTD-S2A(5-16),CTDΔ(17-29)(CTD-S2A(r5-r16),delta(r17-r29))](https://dev.pombase.org/genotype/rpb1-CTD-S2A%285-16%29,CTDdelta%2817-29%29-CTD-S2A%28r5-r16%29,delta%28r17-r29%29-amino_acid_deletion_and_mutation-expression-not_assayed_or_wild_type)

**Comment by kimrutherford** [comment]
OK, we should have a chat about this sometime and decide which of these to show:
https://www.pombase.org/gene_alleles/SPBC28F2.12
